src/alchemtest/namd/idws/descr.rst

```diff
@@ -0,0 +1,33 @@
+NAMD: free energy of dummy ethane to ethane "mutation" in aqueous solution
+==========================================================================
+
+Free energy change from mutating an ethane molecule into an ethane molecule, 
+turning a H atom into a methyl group and conversely.
+Expected free energy is zero, however the dataset is tiny (sufficient for
+testing purposes).
+Uses Interleaved Double-Wide Sampling (Hénin and Brannigan).
+
+Notes
+-----
+Data Set Characteristics:
+    :Number of Legs: 1 (forward mutation in water with IDWS sampling)
+    :Number of Windows: 11
+    :Length of Windows: 500 fs (each window interspersed with 100 fs equilibration)
+    :System Size: 1030 atoms
+    :Temperature: 300 K
+    :Pressure: 1 bar
+    :Alchemical Pathway: dummy mutation of ethane into ethane using dual topology
+                         hybrid molecule. Nonbonded interactions of perturbed
+                         atoms are scaled with their environment.
+    :Experimental Free Energy difference: N/A 
```

Comment:
I agree that "experimental" makes no sense. We don't really control the meta data in a rigorous fashion so I suggest you replace it with
```suggestion
    :Theoretical Free Energy difference: 0
```
which will be useful to anyone reading it. It's an exact theoretical number.